static/css/style.css

```diff
@@ -577,6 +577,9 @@ div.load-form {
 input:checked + .menu-switch-slider {
     background-color: #336699;
 }
+.menu-switch-input:focus~.menu-switch-slider {
+    box-shadow: 0px 0px 5px 3px #FFCC33;
```

Comment:
It might be preferable to use the same focus ring colour for the options as for the other buttons eg light blue rather than yellow so it's consistent and contrasting.